bgpd/bgp_bmp.c

```diff
@@ -350,6 +350,18 @@ static void bmp_put_info_tlv(struct stream *s, uint16_t type,
 	stream_put(s, string, len);
 }
 
+static void bmp_put_vrftablename_info_tlv(struct stream *s, struct bmp* bmp) {
+
+#define BMP_INFO_TYPE_VRFTABLENAME	3
+	char *vrftablename = "global";
+	if (bmp->targets->bgp->inst_type != BGP_INSTANCE_TYPE_DEFAULT) {
+		struct vrf* vrf = vrf_lookup_by_id(bmp->targets->bgp->vrf_id);
+		vrftablename = vrf ? vrf->name : NULL;
+	}
+	if (vrftablename != NULL)
+		bmp_put_info_tlv(s, BMP_INFO_TYPE_VRFTABLENAME, vrftablename);
+}
```

Comment:
commit log not consistent with what the code is doing.
do we want to write "unknown" ?

Reply:
this commit log is a result from 2 locally squashed commits
the first of those commits was getting the name using vrf_id_to_name which was problematic because it outputted "Unknown" when not found so the impl. the resulting commit does a manual name lookup using vrf_lookup_by_id

do you want me to change the commit log ? i guess everything in the PR will be squashed at the end anyways so idk if it's a big deal

Reply:
Thanks for not displaying "unknown". It is ok for me.
Actually, If I read the RFC:
```
   The Information field contains a UTF-8 string whose value MUST be
   equal to the value of the VRF or table name (e.g., RD instance name)
```
be aware that this value will be different from the one introduced in the RD, unless you explicitly configure VRF by using the RD semantic.


That piece of code 'bmp_put_vrftablename_info_tlv()' is unused.
Why keeping  it  if not used?
is there a missing commit ?